spec/javascripts/directives/scheduler/updateDropdownForFilter_spec.js

```diff
@@ -6,17 +6,17 @@ describe('update-drop-down-for-filter initialization', function() {
     var element = angular.element(
       '<form name="angularForm">' +
       '<select id="filter_typ" name="filter_typ" ng-model="scheduleModel.filter_typ"><option>All VMs</option><option>Ketchup</option><option>Relish</option></select>' +
-      '<select id="filter_value" name="filter_value" update-dropdown-for-filter dropdown-model="scheduleModel" dropdown-list="filterList" filter-hide="filterValuesEmpty" ng-model="scheduleModel.filter_value"><option value="" label="Choose">Choose</option><option value="0" label="Mustard">Mustard</option><option value="1" label="Ketchup">Ketchup</option><option value="2" label="Relish">Relish</option>' +
+      '<select id="filter_value" name="filter_value" update-dropdown-for-filter dropdown-model="scheduleModel" dropdown-list="filterList" filter-hide="filterValuesEmpty" ng-model="scheduleModel.filter_value"><option disabled="" value="" class="" selected="selected">Choose</option><option value="string:Mustard" label="Mustard">Mustard</option><option value="string:Ketchup" label="Ketchup">Ketchup</option><option value="string:Relish" label="Relish">Relish</option>' +
```

Comment:
`<option value="string:Ketchup" label="Ketchup">` .. I can't find any reference to this syntax, @AparnaKarve, are you sure that's a thing? I'm pretty sure it should be just `value="Ketchup"`.